pom.xml

```diff
@@ -340,24 +341,6 @@
         </executions>
       </plugin>
 
-      <!-- This plugin cleans up the jars for anything that might cause problems for reproducible builds -->
-      <!--
-        Even if the core maven plugins currently would support doing this without, we would be required
-        to set the line-separator and the Antlr3 plugin crashes if this is set to something non-default
-      -->
-      <plugin>
-        <groupId>io.github.zlika</groupId>
-        <artifactId>reproducible-build-maven-plugin</artifactId>
-        <version>0.11</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>strip-jar</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-
       <!-- Plugin to detect problems with JDK incompatibilities -->

```

Comment:
One of our requirements is that the binaries built on Windows match binaries built on Mac.  The documentation at https://maven.apache.org/guides/mini/guide-reproducible-builds.html seems to warn that it will not change line-endings.  The zlika plugin has made it possible to have Windows and Mac binaries match.

Reply:
> The zlika plugin has made it possible to have Windows and Mac binaries match.

AFAIK, it does not work: did you really check the results?
I tried to rebuild royale-compiler on my Linux box, comparing every output jar file with reference on Maven central with buildinfo:buildinfo automation, and I get many artifacts that are different (reference release has visibly been done on Windows)

Reply:
We have a Windows CI server that builds the artifacts and the 10 jars we care about exactly match on my Mac, but we only verify the actual jar and not the -sources.jar or the source-release.zip.  In your "ko" list, you list two "actual" jars (compiler-0.9.7.jar, debugger-0.9.7.jar).

We do not support Linux at this time, so none of us have tried to verify that Linux matches Win or OSX.  But if you can provide more detail as to what the differences are in those two jars I would find that interesting.  You filed another issue that made me wonder if you are starting from the right sources.  There is a dependency on a jar that helps order output `<compiler-build-tools.version>1.2.0</compiler-build-tools.version>`.  The older 1.1.0 version does not guarantee order.  And we might still have bugs.  All of the recent bugs have had to do with iterator order when iterating through a Java HashMap.  We have not systematically gone and visited every HashMap in the code to enforce order on it, we just fix them as we find them.

I guess I'll try to find time to test without zlika, but they specifically solved one problem for us so we might still need it.

Reply:
the result you see on https://github.com/jvm-repo-rebuild/reproducible-central is done with crlf tweaking on the rebuilder process: if you want I can configure with normal lf and you'll see the additionnal issues
or you can build for yourself: it's just `./rebuild.sh content/org/apache/royale/compiler/royale-compiler-0.9.7.buildspec` with Docker available on your machine: the advantage is that you'll be able to diffoscope to see the actual differences
and to see the result without the crlf workaround, edit the spec to replace `newline=crlf` with `newline=lf`

Reply:
Can you attach the compiler-0.9.7.jar you built from Windows?  I think I followed the instructions and used Docker (for the first time ever).  The results I got had a difference in the META-INF/DEPENDENCIES files and in the generated code from Antlr and JFlex.

Can I specify a specific version of Java 8 such as 1.8.0_201?